adam-assembly/pom.xml

```diff
@@ -51,6 +58,12 @@
               </excludes>
             </filter>
           </filters>
+          <relocations>
+            <relocation>
+              <pattern>org.apache.parquet.avro</pattern>
+              <shadedPattern>org.bdgenomics.adam.shaded.org.apache.parquet.avro</shadedPattern>
```

Comment:
Whats the shade for again?

Reply:
Mostly, you don't want to know.  😉 

Since `parquet-mr` won't include the pull request with the fix we need for some time, I've had to hack up an older version from source to add the fix, build it in adam, then shade `parquet-avro` to another package, while excluding the version in the external jar that does not contain the fix, with an extension to `maven-shade-plugin` to do so correctly.

Reply:
oh my